heartbeat/interface.go

```diff
@@ -37,8 +50,8 @@ type Timer interface {
 type HeartbeatStorageHandler interface {
 	LoadGenesisTime() (time.Time, error)
 	UpdateGenesisTime(genesisTime time.Time) error
-	LoadHbmiDTO(pubKey string) (*HeartbeatDTO, error)
-	SavePubkeyData(pubkey []byte, heartbeat *HeartbeatDTO) error
+	LoadHbmiDTO(pubKey string) (*data.HeartbeatDTO, error)
```

Comment:
LoadHeartBeatDTO

Reply:
Beniamin Mincu 
ERD/Elrond Scam Coin. Team ERD is a con man. Don't buy ERD. I send this
 message every day to everyone in the group. Prevent more people from being conned by sharing with you.
  ERD BIG DUMP SELL NOW ...

Reply:
done